### ISSUE TITLE
Please support multi masks

### ISSUE THREAD

**Issue by hannalee2** [facebookresearch/sam2#711]
Even though config file in yaml does have option to get multi masks, but in the codes 'multi mask' is not used. 

in base model; 
        """  # Original:
        (
            _,
            _,
            _,
            low_res_masks,
            high_res_masks,
            obj_ptr,
            object_score_logits,
        ) = sam_outputs"""

---> would you mind change this so that we can use the multi mask? 
        (
            low_res_multimasks,
            high_res_multimasks,
            ious,
            low_res_masks,
            high_res_masks,
            obj_ptr,
            object_score_logits,
        ) = sam_outputs

I am trying to use multi mask myself, but I had to modify several other functions, it would be great if you support other function as input of multi masks.
